homeassistant/components/opentherm_gw/services.yaml

```diff
@@ -215,6 +215,33 @@ set_led_mode:
             - "W"
             - "X"
 
+set_max_ch_setpoint:
+  name: Set maximum central heating setpoint
+  description: >
+    Set the maximum central heating setpoint override on the gateway.
+    This command is only available with boilers that support this function.
+  fields:
+    gateway_id:
+      name: Gateway ID
+      description: The gateway_id of the OpenTherm Gateway.
+      required: true
+      example: "opentherm_gateway"
+      selector:
+        text:
+    temperature:
+      name: Temperature
+      description: >
+        The maximum central heating setpoint to set on the gateway.
+        Values between 0 and 90 are accepted, but not all boilers support this range.
```

Comment:
Same as above, why 90?

Reply:
I see. It may be a good idea to describe how to find the actual supported min/max values for the users' boiler in the documentation for the service, i.e. which sensors report those values.

Reply:
Ok. I'll do that.